### PR TITLE
feat(TUX-1178/design system): storybook doc stylesheets

### PR DESCRIPTION
**Pull request by Benrajalu**
**What is the problem this PR is trying to solve?**
We have multiple storybooks under the DS banner and they must share styles. 
Storybook styles can be overriden through stylesheets. 

**What is the chosen solution to this problem?**
This PR aims to create and distribute those stylesheets

**Please check if the PR fulfills these requirements**

- [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Review by jmfrancois — APPROVED** [review]
let's play with circular dependencies :)

File: packages/storybook-docs/README.md [hunk 1]
```diff
@@ -21,9 +21,42 @@ $> yarn
...
+<link
+	href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap"
+	rel="stylesheet"
+/>
+```
+
+### preview.js
+
+```js
+import { addons } from '@storybook/addons';
```
Comment (jsomsanith-tlnd) [inline packages/storybook-docs/README.md:56]:
```suggestion
```

File: packages/storybook-docs/src/about.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="About" />
+
+# About Storybook docs
+
+This is just a test story as well for now.
```
Comment (jsomsanith-tlnd) [inline packages/storybook-docs/src/about.stories.mdx:7]: Remove that as discussed together

File: packages/storybook-docs/.storybook/preview.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import '../dist/globalStyles.min.css';
+
+export const parameters = {
+	options: {
+		storySort: {
+			order: ['Components', 'About', '*'],
```
Comment (jsomsanith-tlnd) [inline packages/storybook-docs/.storybook/preview.js:6]:
```suggestion
			order: ['Components', '*'],
```

File: packages/storybook-docs/src/globalStyles.scss [hunk 4]
```diff
@@ -0,0 +1,373 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
```
Comment (jsomsanith-tlnd) [inline packages/storybook-docs/src/globalStyles.scss:10]: what is this for ? It is overridden for the body just under

Comment (Benrajalu) [reply packages/storybook-docs/src/globalStyles.scss:10]: As discussed, this set the generic value of all fonts, and it becomes the basis for 1rem. 1re = 64.5% of (usually) 16px. 
This trick makes it so 1rem = 10px. 
On the body, you build ont top of that rule by setting a default font size.

File: packages/storybook-docs/.storybook/theme.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import { create } from '@storybook/theming';
+import dictionaryLight from '@talend/design-tokens/lib/light/dictionary';
+
+const t = tokenName => {
+	const token = dictionaryLight.find(t => t.name === tokenName);
+	if (!token) {
+		throw Error('Cannot find token ' + tokenName);
+	}
+	return token.value;
+};
```
[the remark this answers is not in the document]
Comment (Benrajalu) [reply packages/storybook-docs/.storybook/theme.js:10]: Can't import stuff from src into this file, sadly. Storybook does not compile them.

**Review by jsomsanith-tlnd — APPROVED** [review]

**Pull request merged**